test/common/test_executor.cpp

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ *  Point Cloud Library (PCL) - www.pointclouds.org
+ *  Copyright (c) 2014-, Open Perception, Inc.
+ *  Author(s): Shrijit Singh <shrijitsingh99@gmail.com>
+ *
+ */
+
+#include <pcl/test/gtest.h>
+#include <pcl/experimental/executor/executor.h>
+
+using namespace executor;
+
+TEST (Executor, InlineExecutor)
+{
+  inline_executor<blocking_t::always_t> exec;
```

Comment:
Maybe make an alias so `inline_executor` is the type instead of `inline_executor<blocking_t::always_t>` (since it'll always be blocking)

Reply:
I think we can have an alias for all like `omp_executor_default` for other executors. What do you think?

Reply:
Done